presto-main/src/main/java/com/facebook/presto/operator/aggregation/ApproximateCountDistinctAggregation.java

```diff
@@ -127,7 +126,7 @@ public static void input(BooleanDistinctState state, @SqlType(StandardTypes.BOOL
         state.setByte((byte) (state.getByte() | (value ? 1 : 2)));
     }
 
-    private static HyperLogLog getOrCreateHyperLogLog(HyperLogLogState state, double maxStandardError)
+    static HyperLogLog getOrCreateHyperLogLog(HyperLogLogState state, double maxStandardError)
```

Comment:
Maybe extract the helper into a common utils rather than keep it in one particular aggregation function. For example, `HyperLogLogUtils`, `HyperLogLogAggregations`, ...